Lib/test/test_contextlib_async.py

```diff
@@ -0,0 +1,205 @@
+import asyncio
+from contextlib import asynccontextmanager
+from test import support
+import unittest
+
+
+def _async_test(func):
+    """Decorator to turn an async function into a test case."""
+    def wrapper(*args, **kwargs):
+        loop = asyncio.new_event_loop()
```

Comment:
You need to close the loop (1), and make sure it's the default loop (2).

Please rewrite to:

```python
loop = asyncio.new_event_loop()
asyncio.set_event_loop(loop)
try:
    loop.run_until_complete(coro)
finally:
    loop.close()
    asyncio.set_event_loop(None)
```